controllers/mediaController.js

```diff
@@ -0,0 +1,60 @@
+// import Media schema from models
+const { User } = require('../models/mainModels.js');
+// destructured media property of User schema
+const { media } = User;
+
+
+// object contains all mediaController middleware
+const mediaController = {};
+
+
+// middleware to get entire media profile of user
+mediaController.getMedia = (req, res, next) => {
+  // find all media catalog
+  User.findOne({_id:req.query.userId}).exec()
+    .then(response => {
+      // console.log to see if fired
+      console.log("mediaController getMedia fired")
+      // store in local memory
+    res.locals.media = response;
```

Comment:
As discussed, we should store the media property of the returned user document on res.locals.media.

If there is no user document found, can you store an empty object {} on res.locals.media (so the front-end knows nothing was found?). This can still be a 200 status code. (successfully found nothing)

Reply:
This is done, we created a conditional in the .then scope